Rhinoceros_Engine/Convert/ToRhino.cs

```diff
@@ -610,6 +620,46 @@ public static List<object> ToRhino(this BHG.CompositeGeometry geometries)
             return geometries.Elements.Select(x => x.IToRhino()).ToList();
         }
 
+        /***************************************************/
+        /**** Representation  fallback                  ****/
+        /***************************************************/
+
+        public static object ToRhino(this IRender render)
+        {
+            return null;
+        }
+
+        /***************************************************/
+        public static Text3d ToRhino(this RenderText renderText)
```

Comment:
Could you group these public method under a header that reads "Public methods - IRender" to follow the convention for the methods in this file?